### PR TITLE
Added PhoneCard

### PR DESCRIPTION
**Pull request by NIKaragu**
Was modified PhoneCard stucture and changed its styles. Added 2 buttons

### CODE REVIEW

File: src/components/Buttons/Buttons.scss
Comment (k-marchuk) [inline src/components/Buttons/Buttons.scss:1]: When our team had a meeting with Danylo we all agreed that we will use CSS Modules for all new components, as it is described in team project repo. Also we will rewrite all existing components later.
So make corresponding changes for this new component.

**Review by k-marchuk — CHANGES_REQUESTED** [review]

File: src/components/Buttons/Buttons.scss [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+  &--is-active {
+    border: 1px solid #e2e6e9;
+  }
+}
+
+.wide-button {
+  @include wideButtonSizes;
+
+  background-color: #4219d0;
```
Comment (k-marchuk) [inline src/components/Buttons/Buttons.scss]: we have that color in variables $accent-color

File: src/components/Buttons/Buttons.scss [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  font-size: 14px;
+  font-weight: 700;
+  color: #fff;
+  border-radius: 48px;
+  border: none;
+  transition: box-shadow, background-color, 0.1s;
+
+  &:hover {
+    box-shadow: 0px 3px 13px 0px #17203166;
```
Comment (k-marchuk) [inline src/components/Buttons/Buttons.scss]: for this shadow we have $button-hover-shadow. Just change offset-y property from 1px to 3px

File: src/components/Buttons/Buttons.scss [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  &--is-active {
+    border: 1px solid #e2e6e9;
+  }
+}
+
+.wide-button {
+  @include wideButtonSizes;
+
+  background-color: #4219d0;
+  font-size: 14px;
```
Comment (k-marchuk) [inline src/components/Buttons/Buttons.scss]: we have class .button in typograhy.scss for buttons styles

File: src/components/Buttons/Buttons.scss [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  width: 40px;
+
+  &:active {
+    color: $button-content-color-on-click;
+    background-color: #fff;
+    border: $button-border-default;
+  }
+
+  &--is-active {
+    border: 1px solid #e2e6e9;
```
Comment (k-marchuk) [inline src/components/Buttons/Buttons.scss]: check all your colors, we have variables for most of them. Here is $elemnts-color

File: src/components/Buttons/Buttons.scss (hunk 3 above)
Comment (NIKaragu) [reply src/components/Buttons/Buttons.scss]: the class, you mentioned, is incompatible with wide-button class

Comment (k-marchuk) [reply src/components/Buttons/Buttons.scss]: what exactly is incompatible?

**Review by k-marchuk — APPROVED** [review]

**Pull request merged**